sections/dashboard/EligibilityBox/EligibilityMap.ts

```diff
@@ -0,0 +1,2956 @@
+type StringMap = Record<string, boolean>;
+
+const EligibilityMap: StringMap = {
```

Comment:
Quite a big file to be storing in source, do you think it might be worth putting on IPFS or something? Again only something worth considering if permanent. Also probably a `Set` would make more sense here rather than object mapping true.

Reply:
Same as above. It would be a good idea to have in IPFS otherwise, though.

Reply:
Regarding using a set vs a hash table, I'm not sure it really matters in this scenario, since under the hood (and performance-wise), they work identically.

Reply:
It's more just for keeping it a bit cleaner and reducing the bloat of unnecessary `: true`, also seems a more a fitting data type for unique strings with the only function of checking if one exists.